### ISSUE TITLE
Support for VanillaBean shaders

### ISSUE THREAD

**Issue by ruvaldak** [IrisShaders/Iris#832]
I are sad. The shaders I've been working on don't work on Iris u.u

https://github.com/ruvaldak/VanillaBean-shaders

[latest.log](https://github.com/IrisShaders/Iris/files/7110530/latest.log)

**Comment by ruvaldak** [comment]
[latest.log](https://github.com/IrisShaders/Iris/files/7110548/latest.log)

Fresh 1.17.1 instance, Iris compiled from 1.17 branch commit d574f98, Sodium compiled from https://github.com/IrisShaders/sodium-fabric 1.17.x/iris branch commit fd9dcc0

**Comment by coderbot16** [comment]
What do you mean by "don't work"? There's nothing obviously wrong that I can see in the logs.

**Comment by ruvaldak** [comment]
> What do you mean by "don't work"? There's nothing obviously wrong that I can see in the logs.

![2021-09-04_17 37 40](https://user-images.githubusercontent.com/15901769/132108667-99c8ba1e-9b2f-4407-abdf-fa7d0d1da8c1.png)

**Comment by ruvaldak** [comment]
```
#version 120

#extension GL_ARB_shader_texture_lod : enable

uniform mat4 gbufferProjectionInverse;

#ifdef GLSLANG
#extension GL_GOOGLE_include_directive : enable
#endif

varying vec2 texCoord;

uniform sampler2D colortex0;
uniform sampler2D colortex5;
uniform sampler2D depthtex0;
uniform float viewWidth;
uniform float viewHeight;

#include "settings.glsl"

void main()
{
    #ifdef SSAO
    //fast simple small bilateral blur by gri573
    int size = AO_BLUR_SIZE;
    float clarity = AO_BLUR_CLARITY;
    float col = texture2D(colortex5, texCoord).r;
    float weight = AO_BLUR_WEIGHT;

    float col1 = 0.0;
    for(int i = 0; i < size; i++) {
        for(int j = 0; j < size; j++) {
            float col0 = texture2D(colortex5, texCoord + (vec2(i, j) - 0.5 * vec2(size)) / vec2(viewWidth, viewHeight)).r;
            float weight0 = max(1.0 - clarity * length(col0 - col), 0.00001);
            col1 += weight0 * col0;
            weight += weight0;
        }
    }
    col = col1 / weight;

    /*DRAWBUFFERS:0*/
    gl_FragData[0] = vec4(texture2D(colortex0, texCoord).rgb * col,1.0);
    #endif
}
```

Something about this being my deferred1.fsh is leading to Iris being incapable of rendering VanillaBean correctly.

**Comment by ruvaldak** [comment]
Fixed with e7bc1c2ea23b5b867af17b8ba8d1dcfe9c8a93e3
